setup.py

```diff
@@ -14,7 +14,12 @@
     long_description_content_type="text/markdown",
     url="https://github.com/SymbiFlow/python-symbiflow-v2x",
     packages=setuptools.find_packages(),
-    install_requires=['lxml', 'pyjson'],
+    install_requires=[
+        'lxml',
+        'pyjson',
+        'vtr_xml_utils @ \
```

Comment:
This is a `test_requires` right?

Reply:
I decided to add it here, because adding a dependency with link to tests_require was failing. I also tried using link_dependencies, but from what I've read this field is deprecated in the recent versions of setuptools, and it did not work.